metadata/transportation/road_centerlines

```diff
@@ -0,0 +1,84 @@
+# Title
+
+Utah Roads
+
+## ID
+
+a6cf0b81-a99e-4ec3-a9b7-7372ae6a6cbb
+
+## Brief Summary
+
+Statewide road centerline data for Utah.
+
+## Summary
+
+Utah Roads is a multi-purpose statewide roads dataset for cartography, routing, and range-based address location. This dataset is also used as the base geometry for deriving the geospatial representation of UDOT's advanced highway linear referencing system ([ALRS](https://gis.utah.gov/products/sgid/transportation/highway-routes-lrs/)) as well as [UGRC's network analysis datasets](https://gis.utah.gov/products/sgid/transportation/).
+
+## Description
+
+This dataset includes information such as the road name, type, speed limit, and infrastructure such as bike lanes. To see specific information about the data model including field names and descriptions, data types, coded value domains, and data model updates, see the [data model spreadsheet](https://docs.google.com/spreadsheets/d/1jQ_JuRIEtzxj60F0FAGmdu5JrFpfYBbSt3YzzCjxpfI/edit#gid=811360546) and the corresponding tabs for update notes. The color-coded section titled Addressing is primarily maintained by the local stewards, where as the section titled DOT/FHWA is primarily maintained by UDOT. UGRC spatially assigns fields in the section titled Placename/Area, based on current SGID polygon boundaries.
+
+### What is the dataset?
+
+### What is the purpose of the dataset?
+
+The Utah Roads layer plays an important role in active transportation planning in Utah. Recent collaborative efforts between UGRC, [UDOT](https://www.udot.utah.gov/connect/,) and [WFRC](https://wfrc.org/) have led to significant enhancements in the bike and pedestrian data on both the roads and trails/pathways layers. You can contribute to the on-street bike data by letting us know where we have gaps or inaccurate information.
+
+<!--- Is there a contact form people should use when they want to let us know there is a gap in the data? Should we direct them to the office email address? -->
+
+This data model is meant to suggest 'best practices'. It is not the intent that all stewards will change their current data management process to match this standard, but rather that stewards will evaluate their current practices to see how well their data 'translates' to this schema for data sharing and Next Generation 911 preparedness.
+
+<!--- Is it common practice for data stewards to change their data to fit ours?  -->
+
+### What does the dataset represent?
+
+<!--- How did we decide to delineate the individual polylines? I noticed that for many roads there is more than one polyline segment for one named road. -->
```

Comment:
**thought**: i believe divided highways have a positive and negative direction segment.

Reply:
also, the direction of the line (arc) must match the address grid (ie: a "N" road must be pointing north). this comes into play for network routing and address geocoding.

Reply:
I've updated this, let me know what you think!